modules/ROOT/partials/integrations/react-quick-start.adoc

```diff
@@ -23,22 +27,105 @@ npx create-react-app tinymce-react-demo
 cd tinymce-react-demo
 ----
 
-ifeval::["{productSource}" == "package-manager"]
-. Install the `+tinymce+` and `+tinymce-react+` packages and save them to your `+package.json+` with `+--save+`.
+ifeval::["{productSource}" == "cloud"]
+. Install the `+tinymce-react+` package and save it to your `+package.json+` with `+--save+`.
 +
 [source,sh]
 ----
-npm install --save tinymce @tinymce/tinymce-react
+npm install --save @tinymce/tinymce-react
+----
+
+. Using a text editor, open `+./src/App.js+` and replace the contents with:
++
+[source,jsx]
+----
+import React, { useRef } from 'react';
+import { Editor } from '@tinymce/tinymce-react';
+
+export default function App() {
+  const editorRef = useRef(null);
+  const log = () => {
+    if (editorRef.current) {
+      console.log(editorRef.current.getContent());
+    }
+  };
+  return (
+    <>
+      <Editor
+        apiKey='your-api-key'
+        onInit={(evt, editor) => editorRef.current = editor}
```

Comment:
```suggestion
        onInit={ (evt, editor) => editorRef.current = editor }
```
For better readability

Reply:
All react code style guides I could find disagree with this spacing.

Reply:
For example this:
https://dev.to/abrahamlawson/react-style-guide-24pp
Has this example:
```jsx
function ItemList(props) {
  return (
    <ul>
      {props.items.map((item, index) => (
        <Item
          key={item.key}
          onClick={(event) => { doSomethingWith(event, item.name, index); }}
        />
      ))}
    </ul>
  );
}
```